cli/flox/doc/manifest.toml.md

```diff
@@ -462,6 +462,113 @@ Shutdown ::= {
 :   An optional list of systems on which to run this service.
     If omitted, the service is not restricted.
 
+## `[include]`
```

Comment:
TODO:

1. Have another read through this with fresher eyes in the morning
2. Consider pulling out a limitations sections about removing fields and remote environments
3. Not that included environments must be locked and can be updated with `flox include upgrade`